pulumi/config.prod.yaml

```
@@ -308,11 +308,13 @@ resources:
   tb:cloudfront:CloudFrontS3Service:
     frontend:
       service_bucket_name: tb-appointment-prod-frontend
-      # This cert must be hosted in us-east-1; this limitation is imposed by CloudFront
-      certificate_arn: arn:aws:acm:us-east-1:768512802988:certificate/bba8ed51-31ca-4db6-8331-87b6a4be9775
+      # This cert must be hosted in us-east-1; this limitation is imposed by CloudFront. It must
+      # also cover both the normal and short URL domains.
+      certificate_arn: arn:aws:acm:us-east-1:768512802988:certificate/1bf9a946-e840-4c70-a726-19865db25a2f
       distribution:
         aliases:
           - appointment.tb.pro
+          - apmt.tb.pro
```

Hm is this the alias for the short link URLs? If so shouldn't this be:

```suggestion
          - apt.mt
```